src/uct/ib/rc/verbs/rc_verbs_ep.c

```diff
@@ -479,9 +479,13 @@ ucs_status_t uct_rc_verbs_ep_flush(uct_ep_h tl_ep, unsigned flags,
                                                UCT_FLUSH_FLAG_REMOTE),
                     "flush flags CANCEL and REMOTE are mutually exclusive");
 
-    if ((flags & UCT_FLUSH_FLAG_REMOTE) &&
-        uct_rc_ep_is_flush_remote(&ep->super)) {
-        return uct_rc_verbs_ep_flush_remote(ep, comp);
+    if (flags & UCT_FLUSH_FLAG_REMOTE) {
+        UCT_CHECK_PARAM(uct_rc_iface_is_flush_remote(&iface->super),
+                        "iface %p is not configured to process flush(remote) "
+                        "op", iface);
```

Comment:
code duplication with rc_mlx5 and dc_mlx5

Reply:
> code duplication with rc_mlx5 and dc_mlx5

yep, unified into macro